apps/sms/views/shared/js/bootstrap.js

```diff
@@ -1,5 +1,14 @@
-/* global Startup */
+/* global App,
+          Startup */
 
+(function() {
 'use strict';
 
+var shimFrame = document.createElement('iframe');
```

Comment:
Yeah, worth a try, though we need to wait for the iframe load I think - trying... thanks for the tip!